web-api-security/src/main/java/com/epam/learning/backendservices/security/controller/impl/SecretControllerImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.learning.backendservices.security.controller.impl;
+
+import com.epam.learning.backendservices.security.controller.SecretController;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.UUID;
+
+@Controller
+public class SecretControllerImpl implements SecretController {
+
+    public String getSecretPage() {
+        return "secret";
+    }
+
+    public String setSecret(Model model, @RequestParam String secret) {
+        UUID uuid = UUID.randomUUID();
+        String url = "http://localhost8080/secret/" + uuid;
```

Comment:
There is a missing colon between host and port - localhost8080. I think using absolute URL is not a good fit since domain name is a subject to change depending on environment.

Optional: change to relative URL.

Reply:
Done

Reply:
Done